src/utils/__tests__/deepEqual.test.js

```diff
@@ -23,3 +23,32 @@ test('returns true if nested arrays are the same', () => {
 test('returns false if nested arrays are different', () => {
   expect(deepEqual(['a', ['b', ['c']]], ['d', ['e', ['f']]])).toBe(false);
 });
+
+function mockIsEqual(other) {
+  return this.value === other.value;
```

Comment:
@cdock1029 - Looks like the tests are failing due to the eslint `no-invalid-this` rule. I think adding an ignore here for this rule would be fine.